.gitignore

```diff
@@ -15,3 +15,5 @@ results
 
 npm-debug.log
 node_modules
+
+rsvg-convert.exe
```

Comment:
I can, but this makes it easier to build on windows, cuz otherwise the *rsvg-convert.exe* has to be inside the *PATH*. I would prefer to leave it. 

Is there a way to change your mind?